tests/stress/test_stress_routes.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
+
+import math
+import os
+import yaml
+import re
+import requests
+import time
+import logging
+import pytest
+
+from tests.common.helpers.assertions import pytest_assert
+from tests.common.plugins.test_completeness import CompletenessLevel
+from tests.common.utilities import wait_until
+from utils import get_crm_resources, check_queue_status
+
+ALLOW_ROUTES_CHANGE_NUMS = 10
+CRM_POLLING_INTERVAL = 1
+MAX_WAIT_TIME = 120
+
+pytestmark = [
+    pytest.mark.topology('t0')
```

Comment:
1. Why is this test only run on T0?
2. How long does this test take to finish at different completeness level?
3. If it is too long, then we should not let it run by default during nightly test.

Reply:
1. This case can run on T1 or T0, other cases related to neighbor may only run on T0. I will delete this mark. 
2. If I set the completeness level 'debug', it will take with 4 minutes, which announce and withdraw the routes two times. The highest level 'thorough' will announce and withdraw the routes five times now, and it will take around ten minutes.